components/server/src/workspace/config-provider.ts

```diff
@@ -90,7 +164,10 @@ export class ConfigProvider {
                     customConfig = parseResult.config
                     await this.fillInDefaultLocations(customConfig, inferredConfig);
                     if (parseResult.validationErrors) {
-                        log.error(logContext, `Skipping invalid config. Errors are ${parseResult.validationErrors.join(',')}`, { repoCloneUrl: commit.repository.cloneUrl, revision: commit.revision, customConfigString });
+                        const err = new InvalidGitpodYMLError(parseResult.validationErrors);
```

Comment:
:+1:

(it's a bit hard to parse the diff, but I understood that these three changes are the only relevant ones, the rest seems to be a move)

Reply:
Indeed. Also note the demotion of log prio. If this fails here it's a user input issue, not a system error